File: FrontEnd/ntpos_admin_mobile/src/models/orderReturn.ts
import { OrderLineItem, OrderTable } from "./order";

export  interface OrderReturn {
  id: string;
  userID: string;
  orderID: string;
  group: string;
  orderReturnDate: number;
  status: string;
  note: string;
  orderLineItemsReturn: OrderLineItem[];
  tablesReturn:OrderTable [];
}

File: BackEnd/tasks/ntpos-microservices/order-service/src/main/java/fitnlu/ntpos/orderservice/domain/model/OrderReturn.java
package fitnlu.ntpos.orderservice.domain.model;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.util.List;

@Getter
@Setter
@Builder
@ToString
public class OrderReturn {
    private String id;
    private String userID;
    private String group;
    private String orderID;
    private long orderReturnDate;
    private List<OrderProduct> orderLineItemsReturn;
    private List<OrderTable> tableReturn;
    private String status;
    private String note;

}
